docs/topics/migrations.txt

```diff
@@ -276,6 +276,14 @@ class to make it importable::
 Please refer to the notes about :ref:`historical-models` in migrations to see
 the implications that come along.
 
+.. _initial-property:
+
+.. note::
+
+    You may add an ``initial = True`` class property to the ``Migration`` class
+    to have Django apply the "initial migration" logic when the
```

Comment:
Actually, if we want to explain the full logic in the easiest possible manner, how about something like this:

Django will fake a migration if the `--fake-migrations` arg is used, migration tries to create a new table or a new field that already exist, and either of the following conditions are met:

* migration has no dependencies

* migration has `initial = True` class property added to the migration class

[ note that it may also make sense to document the `initial = False` explicit skip of --fake-initial, if you can think of a case when this would be useful ]